src/bvals/cc/bnd_info.cpp

```diff
@@ -296,7 +349,8 @@ BndInfo BndInfo::GetSendBndInfo(std::shared_ptr<MeshBlock> pmb, const NeighborBl
 
 BndInfo BndInfo::GetSetBndInfo(std::shared_ptr<MeshBlock> pmb, const NeighborBlock &nb,
                                std::shared_ptr<CellVariable<Real>> v,
-                               CommBuffer<buf_pool_t<Real>::owner_t> *buf) {
+                               CommBuffer<buf_pool_t<Real>::owner_t> *buf,
+                               const OffsetIndices &of) {
```

Comment:
>  Should it be an unnamed argument?

Yes it can be unnamed.

> Is this only included so that GetSetBndInfo has the same call signature as GetCCRestrictInfo?

Yep

Reply:
Made it an unnamed argument